ibis/backends/dask/execution/selection.py

```diff
@@ -37,7 +37,7 @@ def compute_projection_scalar_expr(
     expr,
     parent,
     data,
-    scope: Scope = None,
+    scope: Scope,
```

Comment:
Any reason to remove the default value, instead of setting the type to `Optional[Scope]`, which seems like a most conservative approach?

Reply:
@datapythonista  The problem wasn't the missing type `Optional`: when a default is `None` the type `Optional` is implicitly added, and mypy doesn't have any problem with it (although explicit is always better than implicit :P )

The problem is the line 52:
```python
    scope = scope.merge_scopes(
```
If `scope` is `None` it will raise an exception.

Reply:
Ah, great, thanka for the info. @icexelloss can you have a quick look at this PR please?